sys-devel/haiku-format/haiku_format-17.0.1.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Code formatter for Haiku coding style"
+DESCRIPTION="Format C/C++ code following the Haiku Coding Guidelines.
+
+This is a superset of clang-format with customizations that implement the coding style followed by \
+Haiku and several applications running on Haiku.
+
+haiku-format looks for a configuration file '.haiku-format', going up the file hierarchy starting \
+with the folder of the file you want to format. This optional configuration file can be used to \
+override any default Haiku style options."
+HOMEPAGE="https://github.com/owenca/haiku-format"
+COPYRIGHT="2003-2023 University of Illinois at Urbana-Champaign
+	2018-2023 Owen Pan"
+LICENSE="Apache v2 with LLVM Exception"
+REVISION="1"
+srcGitRev="29214632fa5f0abb8c6f40ac5c13e6cd26a6ec95"
+SOURCE_URI="https://github.com/owenca/llvm-project/archive/$srcGitRev.tar.gz"
+SOURCE_DIR="llvm-project-$srcGitRev"
+CHECKSUM_SHA256="fa3e3a9edf2e06d0f64b09752e0d6fcff3b9836f70cdbcb62052b849cf628b2a"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	git_haiku_format$secondaryArchSuffix = $portVersion
```

Comment:
```suggestion
```

A single package doesn't usually provide more than one package-level entity.  Providing `cmd:git_haiku_format` should be enough.